### PR TITLE
SSC Updates including mudblazor

### PR DESCRIPTION
**Pull request by gcerik**
- mudblazor in the landing
- new tools page
- new guest view

### CODE REVIEW

File: Datahub.Core/Services/SystemNotificationService.cs [hunk 1]
```diff
@@ -143,29 +143,20 @@ private async Task DoNotifyUsers(IEnumerable<string> userIds)
             }
         }
 
-        private static Expression<Func<SystemNotification, bool>> CreateUserNotificationCondition(string userId, bool unreadOnly)
```
Comment (gcalexk) [inline Datahub.Core/Services/SystemNotificationService.cs:146]: Is there any reason to remove this function? Having it here ensures that it's always the same condition for the count and results, and if it ever needs to be updated it won't get out of sync.

File: WebPortal/Pages/Landing/LandingContent.razor [hunk 2]
```diff
@@ -8,7 +8,7 @@
 @inject NavigationManager NavigationManager
 @inject IOptions<DataProjectsConfiguration> DataProjectsConfiguration
 
-<AeTypography Variant="h4" style="margin-top: 2.5rem">@Localizer["My Places"]</AeTypography>
+<MudText Typo="Typo.h6" Class="mt-12">My Places</MudText>
```
Comment (gcalexk) [inline WebPortal/Pages/Landing/LandingContent.razor:11]: This should stay localized. Same thing in RecentLinks.razor

File: WebPortal/Pages/Landing/RecentLinks.razor [hunk 3]
```diff
@@ -3,7 +3,7 @@
 @inject UserLocationManagerService _userLocationManagerService
 @inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
 
-<AeTypography Variant="h4">@Localizer["Recent Places"]</AeTypography>
+<MudText Typo="Typo.h6" Class="mt-12">Recent Places</MudText>
```
Comment (hperezga) [inline WebPortal/Pages/Landing/RecentLinks.razor:6]: Check the localization here.

**Review by hperezga — APPROVED** [review]

File: Datahub.Core/Services/SystemNotificationService.cs (hunk 1 above)
Comment (ErikApption) [reply Datahub.Core/Services/SystemNotificationService.cs:146]: Ping Yannick on this one - I am not sure about the rationale. It's a private one, not attached to either way. We can rollback that one if necessary.

File: WebPortal/Pages/Landing/LandingContent.razor (hunk 2 above)
Comment (ErikApption) [reply WebPortal/Pages/Landing/LandingContent.razor:11]: thank YOU

**Review by gcnabeel — APPROVED** [review]
Apart from the Tools page (which we can update after) this looks fine.

**Pull request merged**